index.html

```diff
@@ -27,22 +19,74 @@
       </div>
      </div>
     </nav>
-    <div>
      <div class="recipies-and-cook-lessons-logo">
       <img src="assets/images/logo.png"/>
     </div>
-    <h1 class="main-largest-heading">One Step to Making a Good Start</h1>
-    <p class="main-description">
+    <div class="recipies-and-cook-lessons-heading">
+     <h1 class="main-largest-heading">One Step to Making a Good Start</h1>
+     <p class="main-description">
      <span>Mauris fermentum tortor non enim aliquet condimentum. Nam aliquam pretium feugiat. Duis</span>
      <span>sem est, viverra eu interdum ac, suscipit nec mauris. Suspendisse commodo tempor sagittis!</span>
      <span>In justo est, sollicitudin eu scelerisque pretium, placerat eget elit.</span>
     </p>
+    </div>
     <div class="recipies-and-cook-lessons-item">
      <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 490.677 490.677" x="0px" y="0px" id="svg-536a" style="height:48px; width:48px;"><g><g><path d="M489.272,37.339c-1.92-3.307-5.44-5.333-9.259-5.333H10.68c-3.819,0-7.339,2.027-9.259,5.333    c-1.899,3.307-1.899,7.36,0.021,10.667l234.667,405.333c1.899,3.307,5.419,5.333,9.237,5.333s7.339-2.027,9.237-5.333    L489.251,48.005C491.149,44.72,491.149,40.645,489.272,37.339z"></path>
      </g></g>
      </svg>
     </div>
+   </div>
   </header>
+  <section class="our-menu-wrapper">
+   <div class="inner-wrapper">
```

Comment:
Zrobić w środku dodatkowy element wrapujący img i heading i wyśrodkować flexem

Reply:
Poprawione

Reply:
Wrapper zle zobaczyłem dodałem do sekcji our-menu przeoczyłem że ma być w headerze jest poprawione zrobione porządnie. Jeżeli uważasz że dalej jest na odpierdol to już nie wiem jak ma być to zrobione.